releases/v20.2.0-beta.1.md

```diff
@@ -0,0 +1,333 @@
+- Renaming and dropping databases now satisfy the same transactional guarantees as online schema changes on tables: Changes will become visible if and only if the transaction commits, and a successful result indicates changes having propagated to the entire cluster. This fixes some long-standing problems with inconsistent database state being visible after renames and drops. (As a side effect of coherent caching for databases, database names can no longer be recycled in multiple schema changes in the same transaction.) [#52975][#52975]
+- The [vectorized execution engine](../v20.2/vectorized-execution.html) now natively supports `DISTINCT` and `FILTER`ing hash aggregation. [#50721][#50721]
+- Added support for `ALTER DATABASE OWNER TO`. This command changes the owner of a database. The user must be an owner of the database to run the command. The user must also be a member of the new owner role directly or indirectly. [#52736][#52736]
+- [`UPSERT`](../v20.2/upsert.html) statements now acquire locks using the `FOR UPDATE` locking mode during their initial row scan, which improves performance for contended workloads. This behavior is configurable using the `enable_implicit_select_for_update` session variable and the `sql.defaults.implicit_select_for_update.enabled` [cluster setting](../v20.2/cluster-settings.html). [#53132][#53132]
+- Users can now convert existing databases into schemas under other databases through the `ALTER DATABASE ... CONVERT TO SCHEMA UNDER PARENT ...` command. This command can only be run by `admin` and is only valid for databases that don't already have any child schemas other than `public`. [#52997][#52997]
+- Implemented the `IdleInTransactionSessionTimeout` variable to allow terminating sessions that are idle in a transaction past the provided threshold. Set the variable by using `SET idle_in_transaction_session_timeout = 'time'`. Sessions that are idle in `OPEN`, `ABORTED`, and `DONE(COMMITWAIT)` transaction states will be terminated if the user idles longer than the threshold time. [#52938][#52938]
+- Non-admin users with `CREATEROLE` are no longer permitted to drop users with the admin role. [#52881][#52881]
+- Users can now specify which subdirectory they wish to write a [`BACKUP`](../v20.2/backup.html) to, via `BACKUP INTO 'subdir' IN x`. This will write `BACKUP` data to `x/subdir/`. [#53139][#53139]
+- Implemented the geometry [builtin](../v20.2/functions-and-operators.html#built-in-functions) `ST_CoordDim` as an alias for `ST_NDims`. [#53193][#53193]
+- Added support for `ALTER SCHEMA <schema> OWNER TO <owner>`. The command changes the owner of a schema. To use the conditions, the following conditions must be met: The user executing the command must be the of the schema. The user executing the command must be a member of the new owner role. The new owner role must have `CREATE` on the database the schema belongs to. [#52781][#52781]
```

Comment:
nit (Missing word): The user executing the command must be the of the schema > The user executing the command must be the owner of the schema.